### PR TITLE
Update Get-DtcTransaction.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue - "fill in the blank" assignments - Description has been updated now.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Agree with @dariomws306, otherwise looks good.

File: docset/winserver2012r2-ps/msdtc/Get-DtcTransaction.md [hunk 1]
```diff
@@ -50,7 +50,14 @@ This command queries the transactions handled by the local DTC instance.
 ## PARAMETERS
 
 ### -AsJob
-{{Fill AsJob Description}}
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to complete. 
+
+The cmdlet immediately returns an object that represents the job and then displays the command prompt. 
+You can continue to work in the session while the job completes. 
+To manage the job, use the `*-Job` cmdlets. 
+To get the job results, use the [Receive-Job](http://go.microsoft.com/fwlink/?LinkID=113372) cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/msdtc/Get-DtcTransaction.md]:
```suggestion
To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
```

File: docset/winserver2012r2-ps/msdtc/Get-DtcTransaction.md [hunk 2]
```diff
@@ -50,7 +50,14 @@ This command queries the transactions handled by the local DTC instance.
...
 ### -AsJob
-{{Fill AsJob Description}}
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to complete. 
+
+The cmdlet immediately returns an object that represents the job and then displays the command prompt. 
+You can continue to work in the session while the job completes. 
+To manage the job, use the `*-Job` cmdlets. 
+To get the job results, use the [Receive-Job](http://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+
+For more information about Windows PowerShell background jobs, see [about_Jobs](http://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/msdtc/Get-DtcTransaction.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

**Comment by sowjanya365** [comment]
> Please use https instead of http. Thanks!

Thank you for the inputs, @dariomws306 @tseward 

Updated all the articles with the suggested changes. Kindly review. Thank you! :)

**Comment by v-raaba** [comment]
@sowjanya365 @e0i Closing this PR please PR against Master, thanks.

**Pull request closed without merging**